packages/flutter/lib/src/rendering/list_wheel_viewport.dart

```diff
@@ -15,8 +15,37 @@ import 'viewport_offset.dart';
 
 typedef double _ChildSizingFunction(RenderBox child);
 
+/// A delegate used by [RenderListWheelViewport] to manage its children.
+///
+/// [RenderListWheelViewport] during layout will asked the delegate to create
+/// children that are visible in the viewport and remove those that are not.
+abstract class ListWheelChildManager {
+  /// The number of children that will be provided.
+  ///
+  /// If non-null, the children will have index in the range [0, childCount - 1].
+  ///
+  /// If null, then there's no explicit limits to the range of the children
+  /// except that it has to be contiguous. If [childExistsAt] for a certain
+  /// index return false, that means the index is either a lower or upper limit.
+  int get childCount;
+
+  /// Check whether the delegate is able to provide a child widget at the given
+  /// index (this function is not about whether the child at the given index is
+  /// active or not).
```

Comment:
You might want to add a clause to make it clear that this actually ends up building that child too.